src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -540,21 +540,16 @@ public static void exitGameIfFinished() {
    * todo, replace with something better
    * Get the chat for the game, or null if there is no chat.
    */
-  public static Chat getChat() {
+  public static Optional<Chat> getChat() {
     final ISetupPanel model = setupPanelModel.getPanel();
     if (model instanceof ServerSetupPanel) {
-      return model.getChatPanel().getChat();
+      return Optional.of(model.getChatPanel().getChat());
     } else if (model instanceof ClientSetupPanel) {
-      return model.getChatPanel().getChat();
+      return Optional.of(model.getChatPanel().getChat());
```

Comment:
Is there a guarantee that getChat() doesn't return null?

Reply:
Tracing it down, looks like it should never be null, but we do not have the guarantee. After `shutdown`, for example, chat would be null, but we do not expect it. For now, it's easy enough to use `ofNullable`, I updated that in: 996834792